aaf_monitor_logs/package.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0"?>
+<package>
+  <name>aaf_monitor_logs</name>
+  <version>0.1.20</version>
+  <description>A package to support MongoDB-based storage and analysis for data from a ROS system, eg. saved messages, configurations etc</description>
+
+  <maintainer email="cburbridge@gmail.com">Chris Burbridge</maintainer>
```

Comment:
Also all of this looks like wrong copy/paste. Can @nilsbore please fix this first?